File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/CPL3010U00LaySpecimenInfoListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
	public class CPL3010U00LaySpecimenInfoListItemInfo
	{
		private String _bunho;
		private String _suname;
		private String _sex;
		private String _birth;
		private String _gwaName;
		private String _doctorName;
		private String _hoDongName;
		private String _hoCode;
		private String _orderDate;
		private String _jubsuDate;
		private String _partJubsuDate;
		private String _jubsuTime;
		private String _partJubsuTime;
		private String _jubsuja;
		private String _inOutGubun;
		private String _jundalGubun;
		private String _specimenCode;
		private String _specimenName;
		private String _tat1;
		private String _tat2;
		private String _age;

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String Suname
		{
			get { return this._suname; }
			set { this._suname = value; }
		}

		public String Sex
		{
			get { return this._sex; }
			set { this._sex = value; }
		}

		public String Birth
		{
			get { return this._birth; }
			set { this._birth = value; }
		}

		public String GwaName
		{
			get { return this._gwaName; }
			set { this._gwaName = value; }
		}

		public String DoctorName
		{
			get { return this._doctorName; }
			set { this._doctorName = value; }
		}

		public String HoDongName
		{
			get { return this._hoDongName; }
			set { this._hoDongName = value; }
		}

		public String HoCode
		{
			get { return this._hoCode; }
			set { this._hoCode = value; }
		}

		public String OrderDate
		{
			get { return this._orderDate; }
			set { this._orderDate = value; }
		}

		public String JubsuDate
		{
			get { return this._jubsuDate; }
			set { this._jubsuDate = value; }
		}

		public String PartJubsuDate
		{
			get { return this._partJubsuDate; }
			set { this._partJubsuDate = value; }
		}

		public String JubsuTime
		{
			get { return this._jubsuTime; }
			set { this._jubsuTime = value; }
		}

		public String PartJubsuTime
		{
			get { return this._partJubsuTime; }
			set { this._partJubsuTime = value; }
		}

		public String Jubsuja
		{
			get { return this._jubsuja; }
			set { this._jubsuja = value; }
		}

		public String InOutGubun
		{
			get { return this._inOutGubun; }
			set { this._inOutGubun = value; }
		}

		public String JundalGubun
		{
			get { return this._jundalGubun; }
			set { this._jundalGubun = value; }
		}

		public String SpecimenCode
		{
			get { return this._specimenCode; }
			set { this._specimenCode = value; }
		}

		public String SpecimenName
		{
			get { return this._specimenName; }
			set { this._specimenName = value; }
		}

		public String Tat1
		{
			get { return this._tat1; }
			set { this._tat1 = value; }
		}

		public String Tat2
		{
			get { return this._tat2; }
			set { this._tat2 = value; }
		}

		public String Age
		{
			get { return this._age; }
			set { this._age = value; }
		}

		public CPL3010U00LaySpecimenInfoListItemInfo() { }

		public CPL3010U00LaySpecimenInfoListItemInfo(String bunho, String suname, String sex, String birth, String gwaName, String doctorName, String hoDongName, String hoCode, String orderDate, String jubsuDate, String partJubsuDate, String jubsuTime, String partJubsuTime, String jubsuja, String inOutGubun, String jundalGubun, String specimenCode, String specimenName, String tat1, String tat2, String age)
		{
			this._bunho = bunho;
			this._suname = suname;
			this._sex = sex;
			this._birth = birth;
			this._gwaName = gwaName;
			this._doctorName = doctorName;
			this._hoDongName = hoDongName;
			this._hoCode = hoCode;
			this._orderDate = orderDate;
			this._jubsuDate = jubsuDate;
			this._partJubsuDate = partJubsuDate;
			this._jubsuTime = jubsuTime;
			this._partJubsuTime = partJubsuTime;
			this._jubsuja = jubsuja;
			this._inOutGubun = inOutGubun;
			this._jundalGubun = jundalGubun;
			this._specimenCode = specimenCode;
			this._specimenName = specimenName;
			this._tat1 = tat1;
			this._tat2 = tat2;
			this._age = age;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/CPL3010U00LaySpecimenInfoListItemInfo.java
package nta.med.data.model.ihis.cpls;

import java.sql.Timestamp;

public class CPL3010U00LaySpecimenInfoListItemInfo {
	private String bunho;
    private String suname;
    private String sex;
    private Timestamp birth;
    private String gwaName;
    private String doctorName;
    private String hoDongName;
    private String hoCode;
    private Timestamp orderDate;
    private Timestamp jubsuDate;
    private Timestamp partJubsuDate;
    private String jubsuTime;
    private String partJubsuTime;
    private String jubsuja;
    private String inOutGubun;
    private String jundalGubun;
    private String specimenCode;
    private String specimenName;
    private String tat1;
    private String tat2;
    private Integer age;
	public CPL3010U00LaySpecimenInfoListItemInfo(String bunho, String suname,
			String sex, Timestamp birth, String gwaName, String doctorName,
			String hoDongName, String hoCode, Timestamp orderDate,
			Timestamp jubsuDate, Timestamp partJubsuDate, String jubsuTime,
			String partJubsuTime, String jubsuja, String inOutGubun,
			String jundalGubun, String specimenCode, String specimenName,
			String tat1, String tat2, Integer age) {
		super();
		this.bunho = bunho;
		this.suname = suname;
		this.sex = sex;
		this.birth = birth;
		this.gwaName = gwaName;
		this.doctorName = doctorName;
		this.hoDongName = hoDongName;
		this.hoCode = hoCode;
		this.orderDate = orderDate;
		this.jubsuDate = jubsuDate;
		this.partJubsuDate = partJubsuDate;
		this.jubsuTime = jubsuTime;
		this.partJubsuTime = partJubsuTime;
		this.jubsuja = jubsuja;
		this.inOutGubun = inOutGubun;
		this.jundalGubun = jundalGubun;
		this.specimenCode = specimenCode;
		this.specimenName = specimenName;
		this.tat1 = tat1;
		this.tat2 = tat2;
		this.age = age;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getSex() {
		return sex;
	}
	public void setSex(String sex) {
		this.sex = sex;
	}
	public Timestamp getBirth() {
		return birth;
	}
	public void setBirth(Timestamp birth) {
		this.birth = birth;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getHoDongName() {
		return hoDongName;
	}
	public void setHoDongName(String hoDongName) {
		this.hoDongName = hoDongName;
	}
	public String getHoCode() {
		return hoCode;
	}
	public void setHoCode(String hoCode) {
		this.hoCode = hoCode;
	}
	public Timestamp getOrderDate() {
		return orderDate;
	}
	public void setOrderDate(Timestamp orderDate) {
		this.orderDate = orderDate;
	}
	public Timestamp getJubsuDate() {
		return jubsuDate;
	}
	public void setJubsuDate(Timestamp jubsuDate) {
		this.jubsuDate = jubsuDate;
	}
	public Timestamp getPartJubsuDate() {
		return partJubsuDate;
	}
	public void setPartJubsuDate(Timestamp partJubsuDate) {
		this.partJubsuDate = partJubsuDate;
	}
	public String getJubsuTime() {
		return jubsuTime;
	}
	public void setJubsuTime(String jubsuTime) {
		this.jubsuTime = jubsuTime;
	}
	public String getPartJubsuTime() {
		return partJubsuTime;
	}
	public void setPartJubsuTime(String partJubsuTime) {
		this.partJubsuTime = partJubsuTime;
	}
	public String getJubsuja() {
		return jubsuja;
	}
	public void setJubsuja(String jubsuja) {
		this.jubsuja = jubsuja;
	}
	public String getInOutGubun() {
		return inOutGubun;
	}
	public void setInOutGubun(String inOutGubun) {
		this.inOutGubun = inOutGubun;
	}
	public String getJundalGubun() {
		return jundalGubun;
	}
	public void setJundalGubun(String jundalGubun) {
		this.jundalGubun = jundalGubun;
	}
	public String getSpecimenCode() {
		return specimenCode;
	}
	public void setSpecimenCode(String specimenCode) {
		this.specimenCode = specimenCode;
	}
	public String getSpecimenName() {
		return specimenName;
	}
	public void setSpecimenName(String specimenName) {
		this.specimenName = specimenName;
	}
	public String getTat1() {
		return tat1;
	}
	public void setTat1(String tat1) {
		this.tat1 = tat1;
	}
	public String getTat2() {
		return tat2;
	}
	public void setTat2(String tat2) {
		this.tat2 = tat2;
	}
	public Integer getAge() {
		return age;
	}
	public void setAge(Integer age) {
		this.age = age;
	}
    
}